sdc/tests/test_dataframe.py

```diff
@@ -332,7 +325,7 @@ def test_impl(n):
         self.assertEqual(count_array_REPs(), 0)
         self.assertEqual(count_parfor_REPs(), 0)
 
-    @skip_numba_jit
+    @skip_numba_jit('iadd unsupported')
```

Comment:
what is `iadd`?

Reply:
Actually `np.sum` of `Series` unsupported as I see. `iadd` was mentioned in internal error.

Reply:
do we support `loc` and `iloc` for dataframes?

Reply:
Also no, but the first error of the test run is about `np.sum`. If to exclude call of `np.sum` we get error on `loc`.